uc-crux-llvm/src/UCCrux/LLVM/Shape.hs

```diff
@@ -266,6 +275,7 @@ ppShapeSeekError =
 
 -- | Modify the 'Shape' at a given 'Cursor'
 modifyA ::
+  forall f m a tag inTy atTy.
```

Comment:
Does anything introduced by this change actually require the type variables to be brought into scope?

Reply:
Whoops, that's leftover from an earlier draft! Will remove.